### ISSUE TITLE
Add synonym URLs to redirect to KEA3 website

### ISSUE THREAD

**Issue by KMJ1618** [MaayanLab/KEA3web#59]
Currently, the all-lowercase "kea3" URL version works to direct users to this site:
https://maayanlab.cloud/kea3/

However, the alternatives
https://maayanlab.cloud/KEA3/
https://maayanlab.cloud/Kea3/

...result in 404: Not Found error messages. To make the site more findable & accessible, these two alternative versions of the URL might be added to direct users to this site.

**Comment by maxim-k** [comment]
@u8sand is it easy to implement?

**Comment by u8sand** [comment]
@maxim-k i agree with you that this should just be applied cluster wide. In-fact the fuzzy proxy would already pick up things like this but the fact that you get a tomcat 404 means the kubernetes ingress is case insensitive. Will investigate tomorrow.. it should be easy.. even https://maayanlab.cloud/kEA4 works

**Comment by u8sand** [comment]
So I did look into this, however it's obnoxiously not supported by the cluster ingress so it's not going to happen any time particularly soon, I submitted a feature request in the ingress we use (`kubernetes-nginx`) but an alternative ingress `traefik` does support case sensitive matching, but is not easily swapped..

If you want to support this for this application in a reasonable timescale, I think you'll have to have some kind of custom 404 tomcat page handler that directs you back to the real site, and not depend on this being fixed at the cluster level since it's kind of low priority and risky, if we move clusters again i'll probably be using `traefik` for this and other reasons.
